host_core/test/e2e/echo_test.exs

```diff
@@ -38,7 +38,7 @@ defmodule HostCore.E2E.EchoTest do
     Process.sleep(1000)
 
     :ok =
-      HostCore.LinkdefsManager.put_link_definition(
+      HostCore.Linkdefs.Manager.put_link_definition(
```

Comment:
I'm going to push a quick commit right after this review to change this in the dashboard as well.